### PR TITLE
Doctrine and OpenCollective

### PR DESCRIPTION
**Pull request by beberlei**
Relevant sponsoring pages on the site still need to be updated in s seperate PR or commit

### CODE REVIEW

**Review by malarzm — APPROVED** [review]
🎉

File: source/blog/2022-03-24-opencollective.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+work to them for a small percentage of the raised capital. The fee is a much
+smaller amount than the taxes that we had to pay on the raised money
+previously, so it is a win-win for us.
+
+We want to assure our sponsors that we still plan to make use of the funds
+to further the Doctrine project and we are actively looking to increase our funding
+for these goals:
+
+- Regulalry organize hackathons for Doctrine Core team contributors, including
+  3-4 days of accomodation, food and travel for roughly 10-15 people.
```
Comment (SenseException) [inline source/blog/2022-03-24-opencollective.md:28]:
```suggestion
- Regularly organize hackathons for Doctrine Core team contributors, including
  3-4 days of accommodation, food and travel for roughly 10-15 people.
```

File: source/blog/2022-03-24-opencollective.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+- Infrastructure, servers and hosting.
+
+- Marketing material such as stickers, t-shirts, other small things.
```
Comment (SenseException) [inline source/blog/2022-03-24-opencollective.md:32]:
```suggestion
- Marketing material such as stickers, t-shirts and other small things.
```

**Review by SenseException — CHANGES_REQUESTED** [review]

File: source/blog/2022-03-24-opencollective.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+title: "On Doctrine Sponsoring with the help of OpenCollective"
+authorName: Benjamin Eberlei
+authorEmail: kontakt@beberlei.de
+permalink: /2022/03/24/opencollective.html
+---
+
+To simplify our own administrative burden, we have decided to move over the
+funds to [OpenCollective](https://opencollective.com/doctrine), primarily
+motiviated by the success and positive experience of the new PHP Foundation.
```
Comment (greg0ire) [inline source/blog/2022-03-24-opencollective.md:10]:
```suggestion
motivated by the success and positive experience of the new PHP Foundation.
```

File: source/blog/2022-03-24-opencollective.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+We have started raising money for the Doctrine in 2019 through Patreon and
+Github Sponsors to support the project. It was planned to organize core team
+gettogethers/hackathons, but due to the pandemic we haven't been able to do this.
```
Comment (greg0ire) [inline source/blog/2022-03-24-opencollective.md:14]:
```suggestion
get-togethers/hackathons, but due to the pandemic we haven't been able to do this.
```

**Review by greg0ire — APPROVED** [review]

**Pull request merged**